index.html

```diff
@@ -82,29 +82,22 @@
 <h1><img src="./Samply.png" alt="Samply"><b style="vertical-align:super"> manifest</b></h1>
 
 <h2>Who is the Samply community?</h2>
-<p>The Samply community started as a group of software developers and scientists from research institutions at different locations, founded to build software for scientific purposes, networking and federated research.
+<p>The Samply community started as a group of software developers and scientists from research institutions at different locations. Its purpose is to build software for scientific purposes, networking and federated research.
 As an open source community, it is our goal to provide the Samply software developed during individual projects to all members of the community.</p>
-<p>Originally established within the German Cancer Consortium (DKTK), the Samply initiative's open-source licensed software is used by several European scientific projects and research networks.</p>
+<p>Originally established within the German Cancer Consortium (DKTK), the Samply initiative's open-source licensed software is now used by several European scientific projects and research networks.</p>
 
 <h2>Communication and organisation within the community:</h2>
 <p>All members of the Samply community are expected to comply with the principles and conventions of the Samply manifest.</p>
-<p>Each individual member of the Samply community moreover adheres to the rules of the <a href="https://berlincodeofconduct.org">Berlin Code of Conduct</a>.</p>
-<p>The Samply community is self-organised and self-committed to divide the workload, implement the software requirements and review pull requests.</p>
+<p>Each individual member of the Samply community moreover should adhere to the rules of the <a href="https://berlincodeofconduct.org">Berlin Code of Conduct</a>.</p>
+<p>The Samply community is self-organised and committed to dividing the workload of creating software requirements, writing code and reviewing pull requests.</p>
 <p>An open communication culture prevails. Although we develop and use Samply in a variety of independent research projects, we are committed to communicating ideas, problems and changes openly.</p>
-<p>For discussions, questions, complaints or feature requests, contact the developers via the <a href="mailto:dev@samply.de">Samply mailing list</a>, <a href="https://github.com/samply">Issue Tracker</a> or <a href="https://chat.germanbiobanknode.de">Samply chat</a>.</p>
+<p>For discussions, questions, complaints or feature requests, contact the developers via the <a href="mailto:dev@samply.de">Samply mailing list</a> or <a href="https://chat.germanbiobanknode.de">Samply chat</a>.</p>

```

Comment:
The URL that was referenced in the Manifest simply points to the Samply home page, not to an issue tracker. Happy to insert an issue tracker if you have a valid URL though.

Reply:
I would leave the mailing list, if it works and definitely refer to the issue trackers of the individual projects. I see the chat problematic, because it is a closed system. GitHub has discussion boards for repositories, but nor for organisations. However, we may create a sample project for this.

Reply:
See latest patch, this should address your points.